src/denormalize.ts

```diff
@@ -86,12 +86,12 @@ export function denormalize(
         responseObjectOrNewParentArray =
           (parentObjectOrArray as MutableResponseObjectArray)[
             parentResponseKey as number
-          ] || {};
+          ] || Object.create(null);
```

Comment:
Because some keys in the prototype readonly we need to use an object without the default prototype chain.

Reply:
Should be the same problem in normalize